src/datasources/constants.ts

```diff
@@ -0,0 +1,8 @@
+import { LinkedEventsSource } from '../types/types';
+
+export const basePaths = {
+  [LinkedEventsSource.Linkedevents]: '/linkedevents/v1/',
+  [LinkedEventsSource.Linkedcourses]: '/linkedcourses/v1/',
```

Comment:
The limitation with this approach is that it supports only current production version of LinkedEvents and LinkedCourses. Base url for test LinkedEvents is https://api.hel.fi/linkedevents-test/v1/ and for LinkedCourses https://api.hel.fi/linkedcourses-test/v1/